### DISCUSSION TITLE
How to skip based on data?

### DISCUSSION THREAD

**Discussion by avialonn** [thepirat000/Audit.NET#565] [Q&A] [answered]
Hi there,
Im using the EF provider and I would like to skip saving the row based on data condition.
What is the best practice for doing so?

`   .Map<LOG, MyLog>((_pipe, entity) =>
                        {

                        })
                        .AuditEntityAction<MyLog>((evt, entry, auditEntity) =>
                        {
 
                        }))
                  .IgnoreMatchedProperties(true));` 

Thank you!

**Comment by thepirat000** [comment, marked as answer]
You can return a boolean `false` from your `AuditEntityAction` to indicate you don't want to save the audit:

```c#
 .AuditEntityAction<MyLog>((evt, entry, auditEntity) =>
{
  if (condition) return false;

  return true;
}))
```

**Comment by avialonn** [comment]
Works! Thanks @thepirat000.
